### ISSUE TITLE
Muledump not working, grey screen.

### ISSUE THREAD

**Issue by Phtevenx** [atomizer/muledump#129]
When I finish putting in my email and password and i click on the Muledump HTML ( Google Chrome), It comes up with a grey screen with a box in the top left. I have deleted it and re-downloaded it many times. This is what is looks like : (I changed the password and email in this copy, but in the actual one I checked that my password and email were correct)

// editor with syntax hilighting is recommended (for example, notepad++ or textmate)

accounts = {

// Put your data here as shown below. Don't forget the commas and quotes!
// If your emails or passwords contain single quotes,
// escape them with backslashes () like this: 'pass\'word'

'@yahoo.com.au':'Password12345',

}// don't delete this line!

// how many characters are displayed in each row (within one account)
rowlength = 7

// change to 1 to switch to testing
testing = 0

// change to 1 to enable price display in tooltips
prices = 0

// change to 1 to enable one-click login (run lib/mulelogin.au3 first)
mulelogin = 0

// 0 = use smart layout (fill empty spaces)
// 1 = show account boxes row by row
nomasonry = 0

Please help!!!
Thanks

**Comment by Phtevenx** [comment]
I have googled around and tried all of the solutions I saw online, but none of them fixed it.

**Comment by Nightfirecat** [comment]
What error comes up in your browser console when you load Muledump?

**Comment by Phtevenx** [comment]
accounts.js Failed to load resource: net::ERR_FILE_NOT_FOUND
totals.js:338 Unknown item: 284 TH Honey bottle
totals.js:338 Unknown item: 1769 Gutsbot Pet Stone
totals.js:338 Unknown item: 2871 Melon Liqueur
totals.js:338 Unknown item: 3280 Cranberries
totals.js:338 Unknown item: 3281 Ear of Corn
totals.js:338 Unknown item: 3282 Sliced Yam
totals.js:338 Unknown item: 3283 Pumpkin Pie
totals.js:338 Unknown item: 3286 Thanksgiving Turkey
totals.js:338 Unknown item: 3321 Pet Form Stone
totals.js:338 Unknown item: 3322 Crashy
totals.js:338 Unknown item: 3323 Marid Pet Stone
totals.js:338 Unknown item: 3393 Wooden Box
totals.js:338 Unknown item: 5465 Potion of Attack (SB)
totals.js:338 Unknown item: 5466 Potion of Defense (SB)
totals.js:338 Unknown item: 5467 Potion of Speed (SB)
totals.js:338 Unknown item: 5468 Potion of Vitality (SB)
totals.js:338 Unknown item: 5469 Potion of Wisdom (SB)
totals.js:338 Unknown item: 5470 Potion of Dexterity (SB)
totals.js:338 Unknown item: 5471 Potion of Life (SB)
totals.js:338 Unknown item: 5472 Potion of Mana (SB)
totals.js:338 Unknown item: 24843 Jaco Pet Stone

**Comment by Phtevenx** [comment]
Is this what you mean? I've a noob at this stuff

**Comment by Nightfirecat** [comment]
Yup, that's perfect&mdash;the error basically says that it can't find your `accounts.js` file. Can you double check that you've followed [these steps](https://github.com/atomizer/muledump/wiki/Troubleshooting#it-shows-nothing-what-did-i-do-wrong) from the troubleshooting wiki?

> You failed to rename the file properly because of your file manager. In Windows, try to [enable file name extensions](http://windows.microsoft.com/en-US/windows7/Show-or-hide-file-name-extensions)

**Comment by Phtevenx** [comment]
Um, could you explain how to 'enable file name extensions'?

**Comment by Phtevenx** [comment]
I checked the first three steps and I did all of them. The file's type is java script, if that's what the last one is.

**Comment by Phtevenx** [comment]
No idea what I did different, but when I launched it this time it worked. Thanks so much for helping me anyway!

**Comment by Nightfirecat** [comment]
Hm... Strange. Well, glad it's working now, anyway! Feel free to close this issue. (button is below the comment box)
